packages/engine/src/proto.rs

```diff
@@ -225,6 +225,21 @@ pub struct ExperimentRunBase {
     pub project_base: ProjectBase,
 }
 
+impl ExperimentRunBase {
+    /// Returns `true` if the experiment uses Python init or has any Python behavior.
+    pub fn has_python_init_or_behavior(&self) -> bool {
+        matches!(
+            self.project_base.package_init.initial_state.name,
+            InitialStateName::InitPy
+        ) || self
```

Comment:
Can you change this to not use a match? As it's a unit variant, it's not required.